frontend/src/app/core/services/auth.service.ts

```diff
@@ -15,33 +16,21 @@ export class AuthService {
 
     private userNameInLS = 'userInfo';
 
-    constructor(private httpClient: HttpClient) {
-        // do nothing
-    }
+    constructor(private httpClient: HttpClient, private httpService: HttpInternalService) {}
```

Comment:
Do we still need httpClient here?

Reply:
Yeah, I deleted, but some checks were not successful